grails-app/services/au/org/emii/aatams/detection/DetectionExtractService.groovy

```diff
@@ -80,7 +80,9 @@ class DetectionExtractService extends AbstractStreamingExporterService {
         }
 
         def endTime = System.currentTimeMillis()
-        log.info("Export finished user: ${getUserInfo()}, sqlRows: ${count}, elapsed time (ms): ${endTime - startTime}, query: ${query}")
+        def sqlLine = query.toString().replaceAll("[\n\r]", "");
```

Comment:
Perhaps replace with space to make it a bit more readable - I'm getting

    select *from valid_detectionwhere (  "rxr_project_name" in ('Whale')  and "timestamp" between timestamp '2011-05-17 12:54:00.0' and timestamp '2017-11-30 11:12:00.0')

Reply:
@jonescc True. I was so distracted by the double spaces I didn't see the munted spaces
Code now producing:
```
INFO  DetectionExtractService - Export finished user: unauthenticated, sqlRows: 23, elapsed time (ms): 122, query: select * from valid_detection where "timestamp" between timestamp '2011-05-17 12:54:00.0' and timestamp '2017-11-30 16:07:00.0'

INFO  DetectionExtractService - Export finished user: unauthenticated, sqlRows: 14, elapsed time (ms): 49, query: select * from valid_detection where ( "rxr_project_name" in ('Seal Count') and "timestamp" between timestamp '2011-05-17 12:54:00.0' and timestamp '2017-11-30 16:07:00.0' )
```